Octokit/Clients/IDeploymentStatusClient.cs

```diff
@@ -50,7 +71,18 @@ public interface IDeploymentStatusClient
         /// <param name="name">The name of the repository.</param>
         /// <param name="deploymentId">The id of the deployment.</param>
         /// <param name="newDeploymentStatus"></param>
-        /// <returns></returns>
         Task<DeploymentStatus> Create(string owner, string name, int deploymentId, NewDeploymentStatus newDeploymentStatus);
+
+        /// <summary>
+        /// Creates a new status for the given deployment. Users with push access can create deployment
+        /// statuses for a given deployment.
+        /// </summary>
+        /// <remarks>
+        /// http://developer.github.com/v3/repos/deployments/#create-a-deployment-status
+        /// </remarks>
+        /// <param name="repositoryId">The ID of the repository.</param>
+        /// <param name="deploymentId">The id of the deployment.</param>
+        /// <param name="newDeploymentStatus"></param>
```

Comment:
Let's name this one and the one above. What about something like `A new deployment status to create.` ?